src/components/OngoingCleaningAlert.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Alert, AlertDescription, AlertTitle } from "@/components/ui/alert";
+
+const OngoingCleaningAlert = () => {
+    return (
+        <Alert variant="destructive">
+            <div className="h-4 w-4" />
+            <AlertTitle>Cleaning Alert</AlertTitle>
+            <AlertDescription>
+                Cleaning is currently ongoing or scheduled within the next 24 hours. A notification cannot be set for this location.
```

Comment:
I wonder why the test for this passes - is something fishy, or did I just miss something?

Reply:
I will put the logic of sorting how unsociable hours will effect this on my to do list! Otherwise we are filtering options of notif times being "in the past" away from the drop down.